### PR TITLE
Feature/add eligibility disclaimer

### PR DESCRIPTION
**Pull request by plunkettgoogle**
## Issue

Add an eligibility disclaimer to the questionnaire.

- Addresses #691

## Description

This PR adds a disclaimer as a final page in the eligibility questionnaire before proceeding to viewing the filtered listings. I'm not closing #691 just yet, since I wasn't sure how to handle the AMI calculation page with this change. I took the liberty of changing the "View Listing" text on all the steps to say "Finish" instead, which brings the user to the disclaimer, since "View Listings" wouldn't make sense if it was bringing the user to the disclaimer.

Desktop:
localhost_3000_eligibility_household

Mobile:
localhost_3000_eligibility_household (1)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Click "Do I Qualify?" on the home page, and go through the questionnaire. Clicking "Finish" at any point should bring you to the disclaimer, and then clicking "View Listings" should bring you to the filtered listings view.

Unit tests: `cd sites/public && yarn test:unit-tests`
Cypress tests: `cd sites/public && yarn test:headless --spec 'cypress/integration/eligibility-questionnaire.ts'`

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

File: sites/public/styles/overrides.scss [hunk 1]
```diff
@@ -232,3 +232,14 @@ a.button.is-primary:hover {
...
 }
+
+.eligibility-disclaimer-header {
+  padding: 20px 4rem;
+}
+
+.eligibility-disclaimer-submit-container {
+  text-align: center;
+  @screen sm {
+    padding-bottom: 40px;
```
Comment (GabeBenjamin) [inline sites/public/styles/overrides.scss:243]: I think you might want this to be in the default css and then have padding-bottom: 0 for @screen sm? Basically on mobile I'm seeing that it needs padding but it doesn't at larger sizes

**Review by GabeBenjamin — CHANGES_REQUESTED** [review]

File: sites/public/styles/overrides.scss [hunk 2]
```diff
@@ -232,3 +232,14 @@ a.button.is-primary:hover {
     border-left-width: 0;
   }
 }
+
+.eligibility-disclaimer-header {
+  padding: 20px 4rem;
```
[the remark this answers is not in the document]
Comment (plunkettgoogle) [reply sites/public/styles/overrides.scss]: Done.

File: sites/public/styles/overrides.scss (hunk 1 above)
Comment (plunkettgoogle) [reply sites/public/styles/overrides.scss]: Ya, not sure what I was thinking here.

**Comment by plunkettgoogle** [comment]
@GabeBenjamin, how is this looking now?

**Review by GabeBenjamin — APPROVED** [review]

**Pull request merged**